File: src/code/js/day_25.js
function ifPrime(number){
    if (number == 1) return false;
    for (let i=2; i*i <= number; i++) if (number % i == 0) return false; 
    return true;
}

function processData(input) {
    let n = input.split("\n")
    for(let t = 1; t<n.length ; t++ ){
        if (ifPrime(n[t])) console.log("Prime"); 
        else console.log("Not prime");
    }
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});

File: src/code/python/day_25.py
import math

def ifPrime(number):
    if number == 1: return False
    for i in range(2, int(math.sqrt(number)) + 1): 
        if number % i == 0: return False
    return True

for _ in range(int(input())):
    if ifPrime(int(input())): print("Prime")
    else: print("Not prime")